src/chrome/content/rules/ResearcherID.com.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+	Connection refused:
+		- dl.researcherid.com
+		- ul.researcherid.com
+
+	Connection reset:
+		- ridws.researcherid.com
+
+	Invalid certificate:
+		- origin-eagan-www.researcherid.com
+		- origin-plano-www.researcherid.com
+
+	Timed out:
+		- account-int.researcherid.com
+		- account-qa.researcherid.com
+		- labs-qa.researcherid.com
+		- rid-qa.researcherid.com
+-->
+
+<ruleset name="ResearcherID.com">
+	<target host="researcherid.com" />
+	<target host="www.researcherid.com" />
+	<target host="account.researcherid.com" />
+	<target host="labs.researcherid.com" />
+
```

Comment:
You can add `<securecookie host="^(www|account|labs)\.researcherid\.com$" name=".+" />` to exclude the problematic domains

Reply:
Added, thanks!